pkg/util/github/github.go

```diff
@@ -79,7 +79,7 @@ func Client() *github.Client {
 	return client
 }
 
-var tokenEnvVars = []string{"MINISHIFT_GITHUB_API_TOKEN", "HOMEBREW_GITHUB_API_TOKEN", "GH_TOKEN"}
+var tokenEnvVars = []string{"MINISHIFT_GITHUB_API_TOKEN"}
```

Comment:
I have made the change in the centos_ci.sh file

Reply:
What is wrong with `"GH_TOKEN"`?

Reply:
#2625 was only to remove `HOMEBREW_GITHUB_API_TOKEN`. Not sure why we are removing `"GH_TOKEN"`

Reply:
I think it is fine to remove `GH_TOKEN` also, as this would be easier to communicate if we have only one env variable that contains the token.

Reply:
> I think it is fine to remove GH_TOKEN also, as this would be easier to communicate if we have only one env variable that contains the token.

I disagree as a quick search for `GH_TOKEN` shows that many uses it so I do not think we should remove it.

Reply:
GH_TOKEN is very common term but usually it good that user creates token as per project and project only use a specific token env instead a variety of it.